### ISSUE TITLE
Missed things in the documentation

### ISSUE THREAD

**Issue by Mondonno** [airly-eu/api-docs#8]
**Hi everyone, and the Airly Staff.**
I found some misslandings in the documentation.

* Please add the `locationId` into the location object describing section (the api returns it but the documentation does not tell us this)
* Please list all levels in the index value (helps to create eg. enums of this levels)
* Please add the list of the indexes, standards and values

**Thanks!**

**Comment by maraswrona** [comment]
Hi @Mondonno 

Thanks for your good remarks. 

The functionality around `location` is not yet fully complete. Once this is ready we will surely update the documentation.
Regarding index levels and measurements these are available in `/meta` endpoints. But fair enough, we could also add this to the docs. Will look into that. 

Cheers
Marek

**Comment by Mondonno** [comment]
Hi @maraswrona 
Thanks for quickly response

Looking forward to changes at docs!
If I will find more gaps in the documentation i create a new issue.

Good night.
Mondonno
